src/e2e-test/features/datastore/runtime.feature

```diff
@@ -0,0 +1,145 @@
+    Then Enter input plugin property: "keyAlias" with value: "fieldName"
+    Then Click on the Get Schema button
+    Then Validate "Datastore" plugin properties
+    Then Close the Plugin Properties page
+    And Select Sink plugin: "Datastore" from the plugins list
+    Then Connect plugins: "Datastore" and "Datastore2" to establish connection
+    Then Navigate to the properties page of plugin: "Datastore2"
+    Then Replace input plugin property: "project" with value: "projectId"
+    Then Enter input plugin property: "referenceName" with value: "refName"
+    Then Select dropdown plugin property: "keyType" with option value: "Key literal"
+    Then Enter input plugin property: "keyAlias" with value: "fieldName"
+    Then Enter kind for datastore plugin
+    Then Enter Ancestor for the datastore plugin
+    Then Validate "datastore2" plugin properties
+    Then Close the Plugin Properties page
+    Then Save and Deploy Pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Validate OUT record count is equal to IN record count
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+
+  @DATASTORE_SOURCE_ENTITY
+  Scenario: To verify data is getting transferred from Datastore to Datastore using Ancestor and Custom Key
+    Given Open Datafusion Project to configure pipeline
+    Then Select plugin: "Datastore" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "Datastore"
+    Then Replace input plugin property: "project" with value: "projectId"
+    Then Enter input plugin property: "referenceName" with value: "ReferenceName"
+    Then Enter kind for datastore plugin
+    Then Enter Ancestor for the datastore plugin
+    Then Select dropdown plugin property: "keyType" with option value: "Key literal"
+    Then Enter input plugin property: "keyAlias" with value: "fieldName"
+    Then Click on the Get Schema button
+    Then Validate "Datastore" plugin properties
+    Then Close the Plugin Properties page
+    And Select Sink plugin: "Datastore" from the plugins list
+    Then Connect plugins: "Datastore" and "Datastore2" to establish connection
+    Then Navigate to the properties page of plugin: "Datastore2"
+    Then Replace input plugin property: "project" with value: "projectId"
+    Then Enter input plugin property: "referenceName" with value: "refName"
+    Then Select dropdown plugin property: "keyType" with option value: "Custom name"
+    Then Enter input plugin property: "keyAlias" with value: "fieldName"
+    Then Enter kind for datastore plugin
+    Then Validate "datastore2" plugin properties
+    Then Close the Plugin Properties page
+    Then Save and Deploy Pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Validate OUT record count is equal to IN record count
```

Comment:
why only validating record count and not whole data?

Reply:
Right now we are just going with validate count. We are not validating the whole data.